### PR TITLE
cp-338: Sidebar improvements for mobile screen size

### PR DESCRIPTION
**Pull request by OlesiaTr**
https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/b0c4bc29-ad9f-4816-8640-60d12a737c20

### CODE REVIEW

**Review by v0ldemar01 — DISMISSED** [review]

**Review by danilittaa — DISMISSED** [review]

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by liza-veis — DISMISSED** [review]

**Review by dmytro-kucherenko — DISMISSED** [review]

**Review by vladyslavdobychin — DISMISSED** [review]

**Review by dmytro-kucherenko — DISMISSED** [review]

File: frontend/src/pages/journal/components/journal-sidebar/styles.module.scss [hunk 1]
```diff
@@ -1,30 +1,52 @@
 .info {
   display: flex;
-  justify-content: center;
   align-items: center;
 }
 
-.journal-entry-list {
+.chat-number {
```
Comment (v0ldemar01) [inline frontend/src/pages/journal/components/journal-sidebar/styles.module.scss]: chat naming in journal sidebar component?

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/journal/components/journal-sidebar/styles.module.scss (hunk 1 above)
Comment (OlesiaTr) [reply frontend/src/pages/journal/components/journal-sidebar/styles.module.scss]: @ossavemu  I am more curious why it was added to my PR to begin with?

**Review by HIMDanny — DISMISSED** [review]

**Review by iamAlinaaa — DISMISSED** [review]

File: frontend/src/pages/journal/components/journal-sidebar/styles.module.scss (hunk 1 above)
Comment (iamAlinaaa) [reply frontend/src/pages/journal/components/journal-sidebar/styles.module.scss]: i guess these changes can be should be deleted ?

Comment (OlesiaTr) [reply frontend/src/pages/journal/components/journal-sidebar/styles.module.scss]: Ye, I'll delete them in a bit

**Review by v0ldemar01 — APPROVED** [review]

**Pull request merged**